spec/mailers/devise_mailer_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe NotifyMailer, type: :mailer do
+  
+  describe "user sign up" do
+    let(:user) { FactoryBot.create(:user) }
```

Comment:
Do we want to configure it to include the `FactoryBot` methods in this type of spec too?

Reply:
To clarify - do you mean that the `FactoryBot` declaration can be dropped so the line is:
```ruby
let(:user) { create(:user) }
```
If so - yes please